main.go

```diff
@@ -86,11 +86,15 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var enableOLMDescriptor bool
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&enableOLMDescriptor, "olm-descriptor", false,
```

Comment:
I have doubts with a cli parameters to be used to set this flag.

What about using a config map to store such flag that SBO would look for it at start and used the values from the CM. If it does not exist, SBO would create it automatically with the default values.

This would allow setting this kind of flags at install time without the need to change the actual Deployment objects (like in case of havint a binary parameter for `--olm-descriptors`). The installation would then go like... Create a ConfigMap in the operator's namespace of a particular name with `enableOLMDescriptor: true` data in it (if users want to enable it) or leave it as before to disable it by default.

Reply:
Yeah, I'm not sure I like the interface; a `ConfigMap` probably would work better.  Are they the conventional way of providing configuration options for an operator?

Reply:
Well, it is a conventional (a "cloud-native") way in Kubernetes in general.